suite-native/receive/src/components/ReceiveAddressCard.tsx

```diff
@@ -0,0 +1,38 @@
+import Animated, { Layout } from 'react-native-reanimated';
+
+import { Card, VStack } from '@suite-native/atoms';
+import { AddressQRCode } from '@suite-native/qr-code';
+
+import { UnverifiedAddress } from './UnverifiedAddress';
+
+type ReceiveAddressCardProps = {
+    address: string;
+    isReceiveApproved: boolean;
+    isUnverifiedAddressRevealed: boolean;
+    onShowAddress: () => void;
+};
+
+export const ReceiveAddressCard = ({
+    address,
+    isUnverifiedAddressRevealed,
+    isReceiveApproved,
+    onShowAddress,
+}: ReceiveAddressCardProps) => (
+    <Animated.View layout={Layout}>
+        {/* TODO: display card alert according to design */}
+        {/* https://github.com/trezor/trezor-suite/issues/9776 */}
+        <Card>
+            <VStack spacing="medium" paddingVertical="small">
```

Comment:
Place the VStack inside UnverifiedAddress as it doesn't affect AddressQRCode please.